dists/opendingux/config.miyoomini.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
same comment as above, this should be better moved to dists/miyoo/

Reply:
Done